modules/_labs/color-picker/react/lib/ColorPicker.tsx

```diff
@@ -65,33 +65,69 @@ export interface ColorPickerProps extends React.HTMLAttributes<HTMLDivElement> {
   transformOrigin?: TransformOrigin;
 }
 
-const defaultColors = [
+const defaultColorSet = [
```

Comment:
Kind of a nit here but I renamed this const to be a more accurate depiction of what these colors are